### PR TITLE
PEP8 Cleanup

### PR DESCRIPTION
**Pull request by sarah-koch**
Updated code to comply with PEP8 standards. Changed import statements to use absolute imports. Reorganized import statements, grouped by source and sorted alphabetically within groups. Removed old, unused code. Ensured consistency in the use of single and double quotes in strings.

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
 from flask import Flask
-from .db import db, migrate
-from .routes.planet_routes import planets_bp
+from app.db import db, migrate
+from app.routes.planet_routes import planets_bp
```
Comment (PriyankaKaramchandani) [inline app/__init__.py:3]: Noted these imports changed from relative imports --> absolute imports. Is this the recommended style?

File: app/__init__.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 def create_app(test_config=None):
     app = Flask(__name__)
 
-    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
-    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://postgres:postgres@localhost:5432/solar_system_development'
+    app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+    app.config["SQLALCHEMY_DATABASE_URI"] = "postgresql+psycopg2://postgres:postgres@localhost:5432/solar_system_development"
```
Comment (PriyankaKaramchandani) [inline app/__init__.py:9]: I see.. switched to double quotes. Both single and double quotes work in Python right?

File: app/routes/planet_routes.py [hunk 3]
```diff
@@ -47,7 +25,7 @@ def get_all_planets():
 
     return [planet.to_dict() for planet in planets]
 
-@planets_bp.get('/<planet_id>')
+@planets_bp.get("/<planet_id>")
```
Comment (PriyankaKaramchandani) [inline app/routes/planet_routes.py:28]: Oh i see are we moving to double quotes to maintain consistency?

**Review by PriyankaKaramchandani — APPROVED** [review]
Refactor changes looks good to me! Looks good to merge to main!

**Review by chardayneal — COMMENTED** [review]
Nice, refactors Sarah!! Looks great!

**Pull request merged**